critiquebrainz/frontend/external/musicbrainz_db/place.py

```diff
@@ -24,30 +25,44 @@ def get_place_by_id(mbid):
     key = cache.gen_key(mbid)
     place = cache.get(key)
     if not place:
-        place = _get_place_by_id(
-            mbid, includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
-        )
+        place = fetch_multiple_places(
+            mbids=[mbid],
+            includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
+        ).get(mbid)
     cache.set(key=key, val=place, time=DEFAULT_CACHE_EXPIRATION)
     return place_rel.process(place)
 
 
-def _get_place_by_id(place_id, includes=None):
+def fetch_multiple_places(*, mbids, includes=None):
```

Comment:
In this case `mbids` can actually be made not keyword-only. It's clear how `fetch_multiple_places([<mbid_1>, <mbid_2>])` would work.